endpointmanager/pkg/fetcher/fhirlist.go

```diff
@@ -0,0 +1,61 @@
+package fetcher
+
+// FHIRList implements the Endpoints interface for epic's endpoint list in FHIR
+type FHIRList struct{}
+
+// GetEndpoints takes the list of cerner endpoints and formats it into a ListOfEndpoints
+// Assumed Structure:
+/**
+{ ... entry: [ {
+		fullUrl: URI for resource
+		resource: {
+			...
+			name: <name of the endpoint>
+			managingOrganiation: { display: <text for resource>, reference: <organization name> },
+			address: <FHIR url>,
+		}
+	  }, ...
+] }
+*/
+func (fl FHIRList) GetEndpoints(fhirList []map[string]interface{}, listURL string) ListOfEndpoints {
+	var finalList ListOfEndpoints
+	var innerList []EndpointEntry
+
+	for entry := range fhirList {
+		fhirEntry := EndpointEntry{}
+		if listURL != "" {
+			fhirEntry.ListSource = listURL
+		} else {
+			fhirEntry.ListSource = string(FHIR)
+		}
+
+		resource, ok := fhirList[entry]["resource"].(map[string]interface{})
+		if ok {
```

Comment:
We should handle the case of this erroring out -- once we start consuming lists in this format it will be good to know where and why we are running into parsing errors

Reply:
good point!